packages/electron-publish/src/gitHubPublisher.ts

```diff
@@ -215,6 +215,20 @@ export class GitHubPublisher extends HttpPublisher {
       })
   }
 
+  private doesErrorMeanAlreadyExists(e: any) {
+    let doesErrorMeanAlreadyExists = true
+    doesErrorMeanAlreadyExists = doesErrorMeanAlreadyExists && e.statusCode === 422
+    if (e.description) {
+      const desc = e.description
+      const descIncludesAlreadyExists =
+        (desc.includes("errors") && desc.includes("already_exists")) || (desc.errors && desc.errors.length >= 1 && desc.errors[0].code === "already_exists")
```

Comment:
I'm not sure I understand this:
`desc.includes("errors") && desc.includes("already_exists")`
Are we expecting a string or array for `desc`?

Reply:
the old code seemed to expect an object, (e.g. `description = { ..., errors: [..., ..., ]}`) but the response back from github is a string with some prose, and some JSON style notation that we really can't parse properly, so I support both ways, not really sure what to expect.